tests/kafka.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test } from '@playwright/test';
+import login from '@lib/auth';
+import { config } from '@lib/config';
+import { navigateToKafkaList, deleteKafkaInstance, createKafkaInstance } from '@lib/kafka';
+import { navigateToKafkaTopicsList, createKafkaTopic, deleteKafkaTopic } from '@lib/topic';
+
+const testInstanceName = `test-instance-${config.sessionID}`;
+const testTopicName = `test-topic-${config.sessionID}`;
+
+test.beforeEach(async ({ page }, testInfo) => {
+  await login(page);
+
+  await navigateToKafkaList(page);
+
+  await page.waitForSelector('[role=progressbar]', { state: 'detached', timeout: config.kafkaInstanceCreationTimeout });
+
+  for (const el of await page.locator(`tr >> a`).elementHandles()) {
+    const name = await el.textContent();
+    await deleteKafkaInstance(page, name);
+  }
+});
+
+// test_4kafka.py test_kafka_topic_create
+test('create and delete a Kafka Topic', async ({ page }) => {
+  await createKafkaInstance(page, testInstanceName);
+  await navigateToKafkaTopicsList(page, testInstanceName);
+  await createKafkaTopic(page, testTopicName);
```

Comment:
if I read this correctly this test can be executed in `kas_with_instance` with no need to duplicate the setup and only run there:

```ts
createKafkaTopic
deleteKafkaTopic
```

Reply:
no, I'm saying that this test should go in the `kas_with_instance` file, as the setup is already 100% compatible.

Reply:
@mmusil do you agree with rearrange it like that?

Reply:
I do

Reply:
@paoloantinori we need to organize tests into files to keep some order, also this way it is easier to compare to IQE/Selenium, otherwise the comparison list spreadsheet is going to be harder to read and understand

Reply:
@asanzgom I believe that we should be more openminded on this. We are doing this in order to improve things. We do not want to copy old structures or old mistakes. How about adding a comment on top of the test method to specify what test in IQE/Selenium it replaces? The reference could then be found easily in the repository with the find command. Or we could put a link to other repo in the comment. Anything... Code duplication is always worse IMHO.

Reply:
The new test has already that comment referencing IQE test, but ok will move it to another file

Reply:
@asanzgom this conversation has **NOT** been resolved, please, feel free to let reviewers close themselves non-obviously addressed comments.

*The ask:*
Move this test to the `tests/kas_with_instance.spec.ts` file.

*The reason:*
In the `tests/kas_with_instance.spec.ts` file we are already doing the very same, super costly, setup to have a Kafka instance in the `Ready` state before starting any test.
The Kafka instance is then re-used by all the subsequent tests (in that file) so that we don't have to wait for it multiple times (up to 10 minutes).
Also, but less importantly, the code for doing the preparation is already self-contained in the file and not duplicated.

*The result:*
Having the [`tests/kafka.ts`](https://github.com/redhat-developer/consoledot-e2e/pull/22/files#diff-d3aa1399d67cb933f51b42212b18afd24eac78c2733a3a7635e9cc2a179de1d4R1) file invalidates **ALL** of the reasons why a comment was done in the first place.

Reply:
I am testing the changes and about to commit